### PR TITLE
Added a dropdown button in Recording Screen

### PR DESCRIPTION
**Pull request by mkenlo**
In this PR, I made change from comments in the previous PR. And I refactored the Recording UI to add a dropdown button to pick the target language for translation: I fetch languages items then create a dropDownItem object.

### CODE REVIEW

File: lib/screens/recording_screen.dart [hunk 1]
```diff
@@ -169,13 +201,42 @@ class _RecordingScreenState extends State<RecordingScreen> {
...
+          DropdownButtonHideUnderline(child: _dropDownButton())
         ]));
   }
 
+  Widget _dropDownButton() {
+    return FutureBuilder(
+        future: fetchLanguages("type=local"),
+        builder: (context, snapshot) {
+          if (snapshot.hasError || !snapshot.hasData)
+            return Text(defaultTargetLang);
```
Comment (evindj) [inline lib/screens/recording_screen.dart:216]: defaultTargetLang should be fetched from user preferences.

File: lib/screens/recording_screen.dart [hunk 2]
```diff
@@ -169,13 +201,42 @@ class _RecordingScreenState extends State<RecordingScreen> {
...
           ),
           Expanded(child: Icon(Icons.compare_arrows)),
-          Text(target, style: TextStyle(fontWeight: FontWeight.bold))
+          DropdownButtonHideUnderline(child: _dropDownButton())
         ]));
   }
 
+  Widget _dropDownButton() {
+    return FutureBuilder(
+        future: fetchLanguages("type=local"),
```
Comment (evindj) [inline lib/screens/recording_screen.dart:213]: Handle error.

**Review by evindj — APPROVED** [review]

**Comment by evindj** [comment]
Great work on this PR, please fix comment and merge!

File: lib/screens/recording_screen.dart (hunk 1 above)
Comment (mkenlo) [reply lib/screens/recording_screen.dart:216]: This will be done as the next feature

File: lib/screens/recording_screen.dart (hunk 2 above)
Comment (mkenlo) [reply lib/screens/recording_screen.dart:213]: Already handling the error

**Pull request merged**